v3/src/components/slider/slider-model.ts

```diff
@@ -50,18 +50,25 @@ export const SliderModel = TileContentModel
   .actions(self => ({
     setValue(n: number) {
       // keep value in bounds of axis min and max when thumbnail is dragged
-      if (n < self.axis.min) { n = self.axis.min }
-      else if (n > self.axis.max) { n = self.axis.max }
-      else if (self.multipleOf) {
+      const keepValueInBounds = (num: number) => {
+        if (num < self.axis.min) return self.axis.min
+        else if (num > self.axis.max) return self.axis.max
+        else return num
+      }
+
+      if (self.multipleOf) {
         n = Math.round(n / self.multipleOf) * self.multipleOf
+        n = keepValueInBounds(n)
```

Comment:
I don't think this should hold up this PR, but perhaps it warrants another story. If `multipleOf` is 5, `self.axis.max` is 9, and `value` is >= 8, this will set the value to 9, which is not a multiple of 5. Essentially, this says that the axis bounds constraint takes precedence over the `multipleOf` constraint and as I think about it that same assumption is present in the animation PR. Ultimately, it seems to me that to get this right we need to find a value that satisfies both the `multipleOf` _and_ and the axis bounds constraint. Perhaps @bfinzer has another idea of how these constraints should work together in the meantime that can be described in the next story.